frontend/packages/dev-console/integration-tests/features/vulnerability/install_quay_container_security_operator

```diff
@@ -0,0 +1,20 @@
+Feature: Install quay container security operator
+    As a user I would like to see Image Vulnerability for each Project
+
+Background:
+    Given user is in Admin perspecitve
+
+@regression
+Scenario: Quay container security operator
+   Given user is at Operator Hub page with the header name "OperatorHub"
+   When user searches for "quay container security"
+   And user clicks quay container security operator card on Operator Hub page
+   And user clicks install button present on the right side bar
+   Then quay container security subscription page will be displayed
+
+@regression
+Scenario: Install the quay container security Operator from Operator Hub page
```

Comment:
Installing any operator should be in the operators.feature file.

Reply:
Updated